README.md

```diff
@@ -47,7 +47,7 @@ A "Hello World" program in Solidity is of even less use than in other languages,
 
 ```solidity
 // SPDX-License-Identifier: MIT
-pragma solidity >=0.6.0 <0.9.0;
+pragma solidity ^0.6.0 <=0.9.0;
```

Comment:
This is not the right use of `^`. Since `0.9.0` doesn't exist, the equality part in `<= 0.9.0` does not make sense. We usually bump the version (`0.9.0`) after its release.

Reply:
G thanks